packages/babel-presets/babel-preset-kyt-core/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-var": 0
```

Comment:
maybe this is naive but I figured there's nothing about these presets that really requires them to be used inside kyt, so might as well make them es5 compatible